server/test/services/openapi/v2/Swagger2SchemaGeneratorTest.java

```diff
@@ -0,0 +1,363 @@
+            .appendLine("          type: integer")
+            .appendLine("          required: false")
+            .appendLine(
+                "          description: \"A positive integer. Limits the number of results per"
+                    + " page. If pageSize is larger than CiviForm's maximum page size then the"
+                    + " maximum will be used. The default maximum is 1,000 and is configurable.\"")
+            .appendLine("        - in: query")
+            .appendLine("          name: nextPageToken")
+            .appendLine("          type: string")
+            .appendLine("          required: false")
+            .appendLine(
+                "          description: \"An opaque, alphanumeric identifier for a specific page"
+                    + " of results. When included CiviForm will return a page of results"
+                    + " corresponding to the token.\"")
+            .appendLine("      produces:")
+            .appendLine("        - application/json")
+            .appendLine("      responses:")
+            .appendLine("        \"200\":")
+            .appendLine("          description: For valid requests.")
+            .appendLine("          headers:")
+            .appendLine("            x-next:")
+            .appendLine("              type: string")
+            .appendLine("              description: A link to the next page of responses")
+            .appendLine("          schema:")
+            .appendLine("            $ref: \"#/definitions/result\"")
+            .appendLine("        \"400\":")
+            .appendLine(
+                "          description: Returned if any request parameters fail validation.")
+            .appendLine("        \"401\":")
+            .appendLine(
+                "          description: Returned if the API key is invalid or does not have access"
+                    + " to the program.")
+            .appendLine("      tags:")
+            .appendLine("        - programs")
+            .appendLine("definitions:")
+            .appendLine("  result:")
+            .appendLine("    type: object")
+            .appendLine("    properties:")
+            .appendLine("      payload:")
+            .appendLine("        type: array")
+            .appendLine("        items:")
+            .appendLine("          type: object")
+            .appendLine("          properties:")
+            .appendLine("            applicant_id:")
+            .appendLine("              type: integer")
+            .appendLine("              format: int32")
+            .appendLine("            application:")
+            .appendLine("              type: object")
+            .appendLine("              properties:")
+            .appendLine("                name:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
```

Comment:
**suggestion**: I discuss enums more below, but question_type is effectively an enum

Reply:
see reply on other comment